book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] When adding rental here, remember there is a `many-to-many` relationship between the `Person` and the `Book` through the `Rental`, what we need to do is to make sure that the `Rental` constructor has been modified appropriately so that when we instantiate the `Rental` class here in this method, the rental will be added to the `@rentals` array in both `Book` and `Person`. 
**HINT**
```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```